docs/docs/single-node-network.md

```diff
@@ -70,13 +70,3 @@ for the network.
 To run the network use the following command.
 
     `fetchd start`
-
-## Resetting the network
-
-Often you will want to clear out all the data from the network and start again. To do that in a local network simply
-run the following command:
-
-    `fetchd unsafe-reset-all`
```

Comment:
why remove this? this can be useful to know

Reply:
It doesn't look like this is supported now on latest, when I've tried it it fails on master: 


```
fetchd unsafe-reset-all
Error: unknown command "unsafe-reset-all" for "fetchd"
Run 'fetchd --help' for usage.
```

Reply:
Oh right, cosmos changed that in latest version. New command is `fetchd tendermint unsafe-reset-all`